### PR TITLE
Add Monique Grinstein 

### LINKED ISSUE #220: Add listed mentors to the website - Monique
### Task
Add the following mentors to the website
URL - https://womencodingcommunity.com/mentors

### Mentors to be added
Monique Grinstein

Reference: https://womenwhocode.github.io/london/mentors (helpful to get mentor details)

### Acceptance Criteria
Screenshot showing the mentors

### PR DESCRIPTION
**Pull request by ssssoz**
## Description

- Copied over Monique Grinstein’s information from the [old mentorship page](https://womenwhocode.github.io/london/mentors#mentor-card-64)
- Fixed Madhura Chaganty’s index number in the books list — it was associating Madhura’s book recommendations to Monique’s profile (due to the change in index numbering since the old website). Monique does not have any resources. 
- Idem for Irina Kamalova’s profile

## Change Type
- [x] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [x] Mentor Update
- [ ] Documentation
- [ ] Other

## Related Issue

Closes #220 

### Presentation 

**Old profile**

Screenshot 2024-05-17 at 18 14 58

**New profile**

Screenshot 2024-05-17 at 18 15 46

### Skills

**Old profile**

Screenshot 2024-05-17 at 18 16 02

**New profile**

Screenshot 2024-05-17 at 18 16 44

### Mentees

**Old profile**

Screenshot 2024-05-17 at 18 16 59

**New profile**

Screenshot 2024-05-17 at 18 17 27

### Reviews

**Old profile**

Screenshot 2024-05-17 at 18 17 40

**New profile**

Screenshot 2024-05-17 at 18 18 02

### Resources

**Old profile**

Screenshot 2024-05-17 at 18 18 15

**New profile**

Screenshot 2024-05-17 at 18 18 32

---

#### Irina

**Before**

Screenshot 2024-05-17 at 18 23 37

**After **

See list from old site: https://womenwhocode.github.io/london/mentors#mentor-card-7

Screenshot 2024-05-17 at 18 23 55

#### Madhura

**Before**

Screenshot 2024-05-17 at 18 25 29

**After **

See old profile: https://womenwhocode.github.io/london/mentors#mentor-card-33

Screenshot 2024-05-17 at 18 25 54

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/mentors.yml [hunk 1]
```diff
@@ -1026,4 +1026,33 @@
     extra: |
       Software Development Strategies, Resume Review, Preparation for The Technical Interview
   network:
-    - linkedin: https://www.linkedin.com/in/liliia-rafikova-635328104/
\ No newline at end of file
+    - linkedin: https://www.linkedin.com/in/liliia-rafikova-635328104/
+
+- name: Monique Grinstein
+  disabled: false
+  sort: 200
```
Comment (dricazenck) [inline _data/mentors.yml]:
```suggestion
  sort: 100
```

This mentor has only 2 hours availability, we are sorting on top mentors that are available in the next months and have more hours.

File: _data/mentors.yml [hunk 2]
```diff
@@ -1026,4 +1026,33 @@
...
+  hours: 2
+  position: Software Engineer, The LEGO Group
+  type: ad-hoc
+  index: 33
+  location: London, UK
+  bio: |
+    I work mainly with Backend and AWS. I am keen to help people start their journey in Software Engineering and Cloud Computing. I can share some knowledge from the industry, help you review your CV to apply for a job, and anything else around that. If you’re looking to get AWS certified, I could also help you come up with a study plan, and share tips and study materials. I currently hold two certifications: Cloud Practitioner & Solutions Architect - Associate.
+  image: assets/images/mentors/monique_grinstein.jpeg
+  languages: English, Portuguese, French
+  availability: []
```
Comment (dricazenck) [inline _data/mentors.yml]: I guess she will be available next 2 months at least. 

```suggestion
  availability: [ 6 , 7 ]
```

File: _data/books.yml [hunk 3]
```diff
@@ -43,7 +43,7 @@
     - title: Start With WHY by Simon Sink
       comments: “How great leaders inspire everyone to take action”. But it`s not just about leaders. There`s valuable insight here to help understand yourself, too. I also recommend you look up Simon`s Ted Talks, they`re fun to watch and really on-point.
 
-- index: 33 # mentor id 33 - madhura
+- index: 7 # mentor id 7 - madhura
```
Comment (dricazenck) [inline _data/books.yml:46]: 👍

File: _data/books.yml [hunk 4]
```diff
@@ -8,7 +8,7 @@
     - title: |
         The Culture Map - Breaking Through the Invisible Boundaries of Global Business by Erin Meyer
 
-- index: 7 # mentor id 7 - Irina
+- index: 4 # mentor id 4 - Irina
```
Comment (dricazenck) [inline _data/books.yml:11]: thanks

**Review by dricazenck — APPROVED** [review]

**Comment by ssssoz** [comment]
@dricazenck You’re fast! I just added additional screenshots to show the updates to Irina & Madhura’s profiles since I realised later that Irina’s book recommendations were also erroneous.

File: _data/mentors.yml (hunk 1 above)
Comment (ssssoz) [reply _data/mentors.yml]: Thanks! Didn’t know what this was for exactly :)

File: _data/mentors.yml (hunk 2 above)
Comment (ssssoz) [reply _data/mentors.yml]: @dricazenck what is this one for / how is it used? :)

**Pull request merged**